src/components/BottomSheet/types.ts

```diff
@@ -0,0 +1,25 @@
+import type React from 'react'
+import type {ViewStyle} from 'react-native'
+import type {BottomSheetContextType} from './contexts/BottomSheetProvider'
+
+export interface BottomSheetProps {
```

Comment:
There are lots of type you put inside the component so why this type you create new file?

Reply:
because a type file it's help us clean code in that component

Reply:
We should make it consistent. All the component that we already implemented don't have `type.ts`

Reply:
updated